### PR TITLE
Chore/zones

### PR DESCRIPTION
**Pull request by MildTomato**
## What kind of change does this PR introduce?

Bug fix, feature, docs update, ...

## What is the current behavior?

Please link any relevant issues here.

no multi zones 😢 

## What is the new behavior?

multi zones!

lots of thanks to @tomhanson for updating the `www` site in a previously closed PR

### CODE REVIEW

File: www/next.config.js [hunk 1]
```diff
@@ -1,6 +1,568 @@
...
+          },
+        ],
+      },
+    ]
+  },
+
+  async rewrites() {
+    return [
+      {
+        source: '/:path*',
```
Comment (inian) [inline www/next.config.js:33]: is this required Jonny?

File: www/next.config.js [hunk 2]
```diff
@@ -1,6 +1,568 @@
...
+        destination: `/:path*`,
+      },
+      {
+        source: '/docs',
+        destination: `${process.env.NEXT_PUBLIC_DOCS_URL}`,
```
Comment (inian) [inline www/next.config.js:38]: currently set to https://zones-docs.vercel.app should be changed to docs-rose.vercel.app...or production branch for zones-docs needs to be set to master

**Review by inian — APPROVED** [review]

File: www/next.config.js (hunk 2 above)
Comment (MildTomato) [reply www/next.config.js:38]: probably going to retire the old project (docs-rose.vercel.app) to make switching over easier

File: www/next.config.js (hunk 1 above)
Comment (MildTomato) [reply www/next.config.js:33]: we're following the docs :shrug:

**Pull request merged**